pages/dkp/konvoy/2.2/choose-infrastructure/index.md

```diff
@@ -10,6 +10,6 @@ enterprise: false
 
 When installing DKP Konvoy for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
 
-The infrastructure you select then determines the specific requirements for a successful installation.
+The infrastructure you select then determines the specific requirements for a successful installation. If you have decided to uninstall DKP Konvoy, refer to the same infrastructure documentation you have selected for specific steps to take down your cluster.
 
-The different types of infrastructures supported in Konvoy are listed below:
+The different types of infrastructures supported in DKP Konvoy are listed below:
```

Comment:
Do we want this to be `DKP Konvoy` - or should it just be `DKP`? - honestly don't know what's best: 

```suggestion
The different types of infrastructures supported in DKP are listed below:
```